flink-runtime-web/web-dashboard/src/app/interfaces/job-detail.ts

```diff
@@ -50,6 +50,8 @@ export interface JobDetailInterface {
   'start-time': number;
   'end-time': number;
   duration: number;
+  parallelism: number;
```

Comment:
hmm the `JobDetailsInfo` doesn't seem to contain a `parallelism` field?

Reply:
Ah, true. Done.